This patch skips all tests related to PTYs: they all fail under portage
because we don't have a valid TTY. Skip them. A bing chunk of them are
skipped through "skipper()" under mock_pty. The rest of them are skipped
by simulation windows (tests that don't run under windows don't run under
portage either).
diff --git a/tests/_util.py b/tests/_util.py
index d954946b..27381cfd 100644
--- a/tests/_util.py
+++ b/tests/_util.py
@@ -24,7 +24,7 @@ support = os.path.join(os.path.dirname(__file__), "_support")
 def skip_if_windows(fn):
     @wraps(fn)
     def wrapper(*args, **kwargs):
-        if WINDOWS:
+        if True:
             skip()
         return fn(*args, **kwargs)
 
@@ -190,6 +190,10 @@ def mock_pty(
         return skip_if_windows
 
     def decorator(f):
+        def skipper(*args, **kwargs):
+            skip()
+
+        return skipper
         import fcntl
 
         ioctl_patch = patch("invoke.runners.fcntl.ioctl", wraps=fcntl.ioctl)
